packages/desktop-client/src/components/autocomplete/SavedFilterAutocomplete.js

```diff
@@ -44,6 +44,11 @@ function FilterList({ items, getItemProps, highlightedIndex, embedded }) {
 export default function SavedFilterAutocomplete({ embedded, ...props }) {
   let filters = useFilters() || [];
 
+  // Sort filtered items by alphabetical
+  filters.sort((a, b) =>
+    a.name.trim().localeCompare(b.name.trim(), { ignorePunctuation: true }),
+  );
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would you mind doing the sorting inside of `useFilters`? Then the fix would be available in all the places where this hook is used.

Also: can you make sure to use `useMemo` here? We should sort only once, not on every render cycle